src/main/java/org/isf/visits/manager/VisitManager.java

```diff
@@ -56,6 +60,38 @@
 
 	@Autowired
 	private ApplicationContext applicationContext;
+	
+	/**
+	 * Verify if the object is valid for CRUD and return a list of errors, if any
+	 *
+	 * @param visit
+	 * @throws OHServiceException
+	 */
+	private void validateVisit(Visit visit) throws OHServiceException {
```

Comment:
I believe this must be made `public` as there is a call to this method in the GUI file `InsertVisit.java` at line 328

Reply:
Yep, forgot to push, thanks!